### PR TITLE
[schema] add FAQ to collapsible schema

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_collapsible_list.proto [hunk 1]
```diff
@@ -19,4 +19,5 @@ message CollapsibleListData {
     google.protobuf.BoolValue expand_only_one = 2; // If `true`, only one item can be expanded at a time.
     InitialExpandedItems initial_expanded_items = 3; // Sets which items are expanded when the page loads.
     Direction direction = 4; // The direction of the text in the list. Either left-to-right or right-to-left.
+    google.protobuf.BoolValue faq_enabled = 5; // If `true`, The collapsible item will appear in search results as an FAQ.
```
Comment (lxgreen) [inline packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_collapsible_list.proto:22]: `is_qapage_data` is more correct. The whole feature aims to provide the FAQ schema structured data. The correct [technical term](https://developers.google.com/search/docs/appearance/structured-data/qapage) is `QAPage`

**Review by lxgreen — CHANGES_REQUESTED** [review]
see my comment

**Comment by maimonav** [comment]
@tamrat-bay please add a test case to this such schema, in ricos-converters/collapsible-list converters (from/to tiptap).

**Review by maimonav — APPROVED** [review]
Looks good 🔥 , you're familiar with the [schema changes flow](https://bo.wix.com/pages/wix-ricos/docs/for-developers/schema-changes) right? @tamrat-bay

**Pull request merged**